src/pyfncall.jl

```diff
@@ -99,8 +99,10 @@ macro pycall(ex)
     end
     func = ex.args[1].args[1]
     args, kwargs = ex.args[1].args[2:end], []
-    if isexpr(args[1],:parameters)
-        kwargs, args = args[1], args[2:end]
+    if length(args) > 0
+        if isexpr(args[1],:parameters)
+            kwargs, args = args[1], args[2:end]
+        end
```

Comment:
```suggestion
    if !isempty(args) && isexpr(args[1],:parameters)
        kwargs, args = args[1], args[2:end]
    end
```
Might as well combine the `if` statements, and use `isempty`